tests/foreman/api/test_contentmanagement.py

```diff
@@ -1113,6 +1113,7 @@ def test_positive_sync_kickstart_repo(
         assert len(caps_pkgs)
         assert sat_pkgs == caps_pkgs
 
+    @pytest.mark.skip_if_open("BZ:2148813")
```

Comment:
This BZ seems to affect Skopeo when inspecting Capsules, so can this be skipped only when running skopeo_cmd rather than skipping the entire test?

Reply:
Updated. :+1: